### PR TITLE
Text updates

### PR DESCRIPTION
**Pull request by harshilsharma63**
#### Summary
Updated following-
1. Changed bot name to `UserFeedback`.
2. Updated welcome text.
3. Updated survey text displayed on mobile.

#### Ticket Link
Fixes https://mattermost.atlassian.net/browse/MM-58460

### CODE REVIEW

File: server/app/survey.go [hunk 1]
```diff
@@ -23,6 +23,8 @@ const (
 	surveySentValue = "survey_sent"
 
 	cacheValidityUserTeamFilter = 7200 // 2 hours in seconds
+
+	surveyPostMessage = "Help us improve your experience. Use Mattermost in a web browser or the desktop app to take the survey."
```
Comment (larkox) [inline server/app/survey.go:27]: Do we want this to say "Mattermost" or "the app"?

**Review by larkox — APPROVED** [review]
Just one minor thing. Approving just to speed up things.

**Review by devinbinnie — APPROVED** [review]

File: server/app/survey.go (hunk 1 above)
Comment (larkox) [reply server/app/survey.go:27]: The problem is white labelling. I think we try to avoid any reference to Mattermost in this plugin because of that.

Comment (harshilsharma63) [reply server/app/survey.go:27]: This point was already discussed and this is the final text after all that discussion in the User Survey channel.

**Pull request merged**